content_scripts/CameraVis.js

```diff
@@ -332,10 +346,6 @@ export class CameraVis {
             this.historyMesh.setPoints(this.historyPoints);
         }
 
-        if (this.sceneGraphNode) {
```

Comment:
I believe this section is related to the jitter elimination work I did

Reply:
Thanks for the review! This is helpful. After taking a step back I've noticed how bad the jitter is too and I'll make sure to fix it.